### ISSUE TITLE
Add supplier to product

### ISSUE THREAD

**Issue by BorderITS** [borderitsolutions/amadaa#155]

**Comment by lcabrini** [comment]
We need to analyze this a bit further. If we add a field called supplier, that means that each product has exactly one supplier. But most companies I know will deal with multiple suppliers for the same product. So I think this data model is going to have to be a little more complex.

What are your thoughts?

**Comment by BorderITS** [comment]
ok if that's the case then we will need suppliers table or?

**Comment by lcabrini** [comment]
Yes, we'll need a suppliers table. We probably also need a many-to-many to link suppliers and products. Will sketch it out.

**Comment by BorderITS** [comment]
ok  that's by me
